core/server/worker/src/main/java/alluxio/worker/netty/BlockReadHandler.java

```diff
@@ -232,15 +284,13 @@ private void openBlock(Channel channel) throws Exception {
       channel.writeAndFlush(new RPCProtoMessage(heartbeat));
     } while (retryPolicy.attemptRetry());
     throw new UnavailableException(ExceptionMessage.UFS_BLOCK_ACCESS_TOKEN_UNAVAILABLE
-        .getMessage(request.mId, request.mOpenUfsBlockOptions.getUfsPath()));
+        .getMessage(request.getId(), request.getOpenUfsBlockOptions().getUfsPath()));
   }
 
   @Override
   protected void incrementMetrics(long bytesRead) {
-    Counter counter = ((BlockReadRequestInternal) mRequest).mCounter;
-    if (counter == null) {
-      throw new IllegalStateException("metric counter is null");
-    }
+    Counter counter = getRequest().getCounter();
```

Comment:
Any risk of `getRequest` returning null (applies to most cases of using `getRequest`)